### PR TITLE
i18n(fr): Add `guides/ecommerce.mdx` in French language

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description (required)
Add `guides/ecommerce.mdx` in French language

#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/fr/guides/ecommerce.mdx [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+     window.SnipcartSettings = {
+       publicApiKey: "YOUR_API_KEY",
+       loadStrategy: "on-user-interaction",
+     };
+    
+     (function()...); // disponible dans la documentation de Snipcart
+    </script>
+    ```
+
+2. Personnalisez `window.SnipcartSettings` avec l'un des [paramètres Snipcart disponibles](https://docs.snipcart.com/v3/setup/installation#setting2) pour contrôler le comportement et l'apparence de votre panier.
```
Comment (TheOtterlord) [inline src/content/docs/fr/guides/ecommerce.mdx:132]:
```suggestion
2. Personnalisez `window.SnipcartSettings` avec l'un des [paramètres Snipcart disponibles](https://docs.snipcart.com/v3/setup/installation#settings) pour contrôler le comportement et l'apparence de votre panier.
```

File: src/content/docs/fr/guides/ecommerce.mdx [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+      data-item-custom1-name="Couleur du cadre"
+      data-item-custom1-options="Marron|Argent[+10.00]|Or[+20.00]"
+      data-item-custom2-name="Instructions de livraison"
+      data-item-custom2-type="textarea"
+    >
+      Ajouter au panier
+    </button>
+    ```
+
+4. Ajoutez un bouton de paiement Snipcart avec la classe `snipcart-checout` pour ouvrir le panier et permettre aux visiteurs de terminer leur achat avec un formulaire de paiement.
```
Comment (TheOtterlord) [inline src/content/docs/fr/guides/ecommerce.mdx:167]:
```suggestion
4. Ajoutez un bouton de paiement Snipcart avec la classe `snipcart-checkout` pour ouvrir le panier et permettre aux visiteurs de terminer leur achat avec un formulaire de paiement.
```

**Review by TheOtterlord — COMMENTED** [review]
Thanks! Adding some suggestions porting changes from https://github.com/withastro/docs/pull/7150

**Comment by thomasbnt** [comment]
Thanks @TheOtterlord for two reviews from https://github.com/withastro/docs/pull/7150/files !

**Review by dreyfus92 — APPROVED** [review]
Thank you both, LGTM ✅

**Pull request merged**